rest-api/src/main/java/gov/cms/qpp/conversion/api/model/Metadata.java

```diff
@@ -27,12 +27,18 @@
 	private Boolean overallStatus;
 	private Boolean conversionStatus;
 	private Boolean validationStatus;
-	private Boolean cpc;
+	private String cpc;
 	private String conversionErrorLocator;
 	private String validationErrorLocator;
 	private String rawValidationErrorLocator;
+
+	private Date createdDate;
```

Comment:
Can we not just store this as an Instant?

Reply:
There is no native mapping in the DynamoDB mapper for `Instance` nor `LocalDate` (bah!), but we could make our own custom mapper.